test/blocks/marquee/mocks/body.html

```diff
@@ -315,3 +315,28 @@ <h1 id="standing-out-amid-a-sea-of-labels-1">Standing out amid a sea of labels.<
     </div>
   </div>
 </div>
+
+<h2>Marquee with button overlay</h2>
+<div class="marquee">
+  <div>
+    <div>#000000</div>
+  </div>
+  <div>
+    <div>
+      <h2 id="heading-xl-marquee-standard-medium-left">Heading XL Marquee standard medium left</h2>
+      <p>Body M Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.</p>
+      <p><em><a href="https://www.adobe.com/">Lorem ipsum</a></em> <strong><a href="https://www.adobe.com/">Learn more</a></strong></p>
+    </div>
+    <div>
+      <p>
+        <picture>
+          <source type="image/webp" srcset="./media_12a67eee02f1aa5a0c2f2d2d1d540dbd1ac424b35.png?width=2000&amp;format=webply&amp;optimize=medium" media="(min-width: 600px)">
+          <source type="image/webp" srcset="./media_12a67eee02f1aa5a0c2f2d2d1d540dbd1ac424b35.png?width=750&amp;format=webply&amp;optimize=medium">
+          <source type="image/png" srcset="./media_12a67eee02f1aa5a0c2f2d2d1d540dbd1ac424b35.png?width=2000&amp;format=png&amp;optimize=medium" media="(min-width: 600px)">
+          <img loading="eager" alt="" type="image/png" src="./media_12a67eee02f1aa5a0c2f2d2d1d540dbd1ac424b35.png?width=750&amp;format=png&amp;optimize=medium" width="720" height="520">
```

Comment:
png path mentioned here are giving 404 and hence the test cases are failing
<img width="823" alt="Screen Shot 2023-04-12 at 10 20 36 AM" src="https://user-images.githubusercontent.com/69535463/231536889-8d8921fb-e65f-413b-9491-d05352b7275f.png">

Could you please check. Thanks.

Reply:
Hi @Ruchika4 ,  All of the tests I ran pass locally, I did notice the one that failed was in the utils.js file: 
![Screen Shot 2023-04-12 at 11 13 38 AM](https://user-images.githubusercontent.com/17607967/231547412-a43bfd48-b7d0-4447-9cf2-cccb65efa34a.png)

Also, I did not resolve the network 404s because it does not appear to be an issue that is focused on in  our unit tests:
![Screen Shot 2023-04-12 at 11 10 57 AM](https://user-images.githubusercontent.com/17607967/231547292-6c292230-6f59-442e-9aa4-95ec99fe59bb.png)
Do you think it would be better to remove the 404s in this particular instance? Maybe a good community topic? 

I am happy to look into the utils test, but they are all passing locally for me.

Reply:
Odd with the unit tests. I added the discussion here: https://github.com/orgs/adobecom/discussions/635. Thank you for the input!